gpo-examples/get_order.py

```diff
@@ -0,0 +1,20 @@
+# Get order information
+
+from obtain_token import obtain_session
+
+# Key of the order you want to get
+order_key = "FILL_IN"
+
+session = obtain_session()
+submitter_email = "FILL_IN"
+headers = {
+    "Actor-Email": submitter_email,
+    "Content-Type": "application/json",
+    "Accept": "application/json",
+    "User-Agent": "bcl-example",
+}
+res = session.get(
+    f"https://gpo-staging.broadinstitute.org/api/order/{order_key}", headers=headers
+)
+res.raise_for_status()
```

Comment:
I think it may make sense to put this in a method.

Pass in the order key and submitter email in the method and right before the print call the method.  That way this becomes actually reusable.

Reply:
Done by you 👍